test/index.test.ts

```
@@ -0,0 +1,214 @@
+  const lines = raw.split("\n");
+  const indentLength = Math.min(
+    ...lines
+      .filter((line) => line.trim().length > 0)
+      .map((line) => line.match(/^ */)?.[0].length || 0),
+  );
+
+  return lines
+    .map((line) => line.slice(indentLength))
+    .join("\n")
+    .trim();
+}
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)